nemo/collections/nlp/models/language_modeling/megatron_gpt_prompt_learning_model.py

```diff
@@ -344,7 +373,40 @@ def validation_step(self, batch, batch_idx):
         if loss_mean.item == 0.0:
             loss_mean = []
 
-        return loss_mean
+        if self.cfg.get('report_validation_metric', False):
+            preds_text, labels_text = [], []
+            input_ids, labels, loss_mask, position_ids, attention_mask, taskname_ids = batch
+            input_lenghts = torch.cuda.LongTensor(
```

Comment:
Not sure if this is correct since different items in a batch will have different input lengths but this code will have input length the same in the batch.

Reply:
Please check this again!